provision/initramfs/init

```diff
@@ -215,14 +215,17 @@ ifup() {
     fi
     # If we have gotten here, give up for this device, and bring device down
     # just in case its present.
-    ifconfig $DEVICE down >/dev/null 2>&1
+    ip link set dev $DEVICE down >/dev/null 2>&1
     return 255
 }
 
 
 # Bring up networks
 msg_white "Bringing up local loopback network:"
-if ifconfig lo 127.0.0.1 netmask 255.255.255.0 up >/dev/null 2>&1; then
+if ip link set dev lo up >/dev/null 2>&1; then
+    # If this addr already exists for this interface ip will return
+    # error, don't fail due to that, just ignore return value.
+    ip address add 127.0.0.1/8 dev lo >/dev/null 2>&1
```

Comment:
Can you tack a `|| true` on this line since we don't care if it fails. While we're not using `set -o errexit`, this does ensure if we ever get to that point, it will be safely ignored.